### PR TITLE
Translated generic section of localflavor to Dutch

### PR DESCRIPTION
**Pull request by ramonakira**
Just a few small changes to the Dutch translations to be able to use the IBANField with Dutch texts.

### CODE REVIEW

File: localflavor/locale/nl/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -5152,39 +5152,39 @@ msgstr "Schotland"
...
 
 #: generic/validators.py:185
 msgid "Not a valid IBAN."
-msgstr ""
+msgstr "Geen geldige IBAN."
 
 #: generic/validators.py:205
 msgid "BIC codes have either 8 or 11 characters."
-msgstr ""
+msgstr "BIC codes hebben 8 of 11 tekens."
```
Comment (jieter) [inline localflavor/locale/nl/LC_MESSAGES/django.po]: `BIC-codes` http://www.spatiegebruik.nl/regels.html

File: localflavor/locale/nl/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -5194,11 +5194,11 @@ msgstr ""
 #: generic/validators.py:216
 #, python-format
 msgid "%s is not a valid country code."
-msgstr ""
+msgstr "%s is geen geldige landcode."
 
 #: generic/validators.py:225
 msgid "Not a valid EAN code."
-msgstr ""
+msgstr "Geen valide EAN code."
```
Comment (jieter) [inline localflavor/locale/nl/LC_MESSAGES/django.po]: `EAN-code`

**Comment by jieter** [comment]
While your at it, this doesn't look right: 

```
#: gb/gb_regions.py:190
msgid "Wales"
msgstr "Schotland"
```

**Comment by nagisa** [comment]
Translations are done through [Transifex](https://www.transifex.com/django/django-localflavor/dashboard/)

**Comment by ramonakira** [comment]
Oops. I will add the translations there.

**Comment by ramonakira** [comment]
What does one have to do to be accepted to the translation team on transifex?

**Comment by claudep** [comment]
Being approved by the language coordinator.

**Pull request closed without merging**